justfile

```
@@ -15,7 +15,12 @@ watch:
     @echo "Watching slides"
     typst w slides.typ
 
-# Compile and Run the C code
-c:
-    @echo "Compiling and Running the C code"
-    @echo "TODO!"
\ No newline at end of file
+# Compile the C code in "code/"
+check:
+    #!/usr/bin/env bash
+    echo "compiling all *.c code to output"
```

I prefer to use the `just` builtin `@echo` instead of bash's `echo`
```suggestion
    @echo "compiling all *.c code to output"
```